tmail-backend/jmap/extensions/src/main/java/com/linagora/tmail/james/jmap/ESEmailAddressContactSearchEngine.java

```diff
@@ -42,12 +45,17 @@ public ESEmailAddressContactSearchEngine(ReactorElasticSearchClient client) {
 
     @Override
     public Publisher<BoxedUnit> index(AccountId accountId, EmailAddressContact contact) {
+        ObjectMapper objMp = new ObjectMapper();
```

Comment:
Can we please avoid creating an object mapper for each request?

Reply:
Yes, fixed it by creating a private static final instead.